interactive_engine/bin/giectl.sh

```diff
@@ -0,0 +1,396 @@
+  cp $WORKSPACE/config/coordinator.application.properties $inner_config
+  sed -i "s/ZOOKEEPER_PORT/$2/g" $inner_config
+  pushd $WORKSPACE/coordinator/target/classes/
+  java ${JAVA_OPT} com.alibaba.maxgraph.coordinator.CoordinatorMain $inner_config $1 1>$LOG_DIR/maxgraph-coordinator.out 2>$LOG_DIR/maxgraph-coordinator.err &
+  echo $! > $PID_DIR/coordinator.pid
+  popd
+}
+
+start_frontend() {
+  JAVA_OPT="-server
+            -verbose:gc
+            -Xloggc:${LOG_DIR}/maxgraph-frontend.gc.log
+            -XX:+PrintGCDetails
+            -XX:+PrintGCDateStamps
+            -XX:+PrintHeapAtGC
+            -XX:+PrintTenuringDistribution
+            -Djava.awt.headless=true
+            -Dsun.net.client.defaultConnectTimeout=10000
+            -Dsun.net.client.defaultReadTimeout=30000
+            -XX:+DisableExplicitGC
+            -XX:-OmitStackTraceInFastThrow
+            -XX:+UseG1GC
+            -XX:InitiatingHeapOccupancyPercent=75
+            -Dfile.encoding=UTF-8
+            -Dsun.jnu.encoding=UTF-8
+            -Dlogfilename=${LOG_DIR}/maxgraph-frontend.log
+            -Dlogbasedir=${LOG_DIR}/frontend
+            -Dlog4j.configurationFile=file:$WORKSPACE/config/log4j2.xml
+            -classpath $WORKSPACE/0.0.1-SNAPSHOT/conf/*:$WORKSPACE/0.0.1-SNAPSHOT/lib/*:"
+  REPLACE_SCHEMA_PATH=`echo ${2//\//\\\/}`
+
+  inner_config=$CONFIG_DIR/frontend.vineyard.properties
+  cp $WORKSPACE/config/frontend.vineyard.properties $inner_config
+  sed -i "s/VINEYARD_SCHEMA_PATH/$REPLACE_SCHEMA_PATH/g" $inner_config
+  sed -i "s/ZOOKEEPER_PORT/$3/g" $inner_config
+
+  pushd $WORKSPACE/frontend/frontendservice/target/classes/
+
+  java ${JAVA_OPT} com.alibaba.maxgraph.frontendservice.FrontendServiceMain $inner_config $1 1>$LOG_DIR/maxgraph-frontend.out 2>$LOG_DIR/maxgraph-frontend.err &
+
+  timeout_seconds=60
+  wait_period_seconds=0
+
+  sleep 10
+  while true
+  do
+    gremlin_server_port=$(awk '/frontend host/ { print }' ${LOG_DIR}/maxgraph-frontend.log | awk -F: '{print $6}')
+    if [ -n "$gremlin_server_port" ]; then
+      log "FRONTEND_PORT:127.0.0.1:$gremlin_server_port"
+      break
+    fi
+    wait_period_seconds=$(($wait_period_seconds+5))
+    if [ ${wait_period_seconds} -gt ${timeout_seconds} ];then
+      err "Get external ip of ${GREMLIN_EXPOSE} failed."
+      break
+    fi
+    sleep 5
+  done
+
+  echo $! > $PID_DIR/frontend.pid
+  popd
+}
+
+start_executor() {
+  export VINEYARD_IPC_SOCKET=$3
+  inner_config=$CONFIG_DIR/executor.vineyard.properties
+  cp $WORKSPACE/config/executor.vineyard.properties $inner_config
+  sed -i "s/VINEYARD_OBJECT_ID/$1/g" $inner_config
+  sed -i "s/ZOOKEEPER_PORT/$4/g" $inner_config
+  server_id=1
+
+  flag="maxgraph$1executor"
+  RUST_BACKTRACE=full $WORKSPACE/bin/executor --config $inner_config ${flag} ${server_id} 1>> $LOG_DIR/maxgraph-executor.out 2>> $LOG_DIR/maxgraph-executor.err &
+
+  echo $! > $PID_DIR/executor.pid
+}
+
+create_instance_on_local() {
+  cluster_type=$1
```

Comment:
what do you mean by this parameter?
- cluster_type? while the function name is on_local?
- if you want to use this arg, why there is no this arg in the function below?